frontend/packages/dev-console/locales/en/devconsole.json

```diff
@@ -63,15 +63,15 @@
   " by ": " by ",
   "Commit details not available": "Commit details not available",
   "Cluster URL not available": "Cluster URL not available",
-  "Application Stages": "Application Stages",
-  "Application Details": "Application Details",
+  "Environments": "Environments",
+  "Application Environments": "Application Environments",
```

Comment:
Looks like this string isn't being used. The frontend tests are failing.
Run `i18n` again.